### DISCUSSION TITLE
how can i use this library in flutter

### DISCUSSION THREAD

**Discussion by vishwaprojects** [Reedyuk/blue-falcon#118] [Q&A] [answered]
i want to use this library and use flutter as front end how to achieve this, please help me

**Comment by Reedyuk** [comment, marked as answer]
i would strongly recommend you don't do that, you are merge two tools that achieve similar jobs and a guaranteed headache.

If you must, i would suggest you do something like create a KMM Library which uses bluefalcon and exposes a viewmodel and then you import this library natively into your flutter project.

**Reply by vishwaprojects** [reply]
thank you for your reply @Reedyuk  i am trying to do similar kind of thing ..and i am not understanding where i have to start and how to write libraries and use this libs in flutter for android and ios, would be a great help if you help me

**Reply by Reedyuk** [reply]
i would suggest you try the kotlin lang slack channel https://kotlinlang.org/community/

**Reply by vishwaprojects** [reply]
thank you
